src/posts/hello-day-8.md

```diff
@@ -124,9 +124,11 @@ If this works correctly on publish, it will resolve the last of my base requirem
 
 Ok, I was thinking about how to handle build-time cache-breaking and realized that there's likely a way to handle getting a cache-break variable at the build stage. There's [a plugin for Jekyll to do it](https://github.com/jekyll/github-metadata), it looks like [it does so at least partially via the Github API](https://github.com/jekyll/github-metadata/blob/master/docs/authentication.md). It gets [a pretty good list of data too](https://github.com/jekyll/github-metadata/blob/master/docs/site.github.md). There's also [the "Github Context" which is available to GitHub actions](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context). I could call the API during build time, which is what it appears that Jekyll is doing (I didn't really look too deeply into the plugin). But if this data is available in the Actions context... couldn't I export it as a environment variable? Why not try adding that to the Github Actions script?
 
+{% raw %}
 ```yaml
         - run: export GITHUB_HEAD_SHA=${{ github.run_id }}
 ```
+{% endraw %}
```

Comment:
Ah, I think the `{{ github.run_id }}` was trying to be evaluated by Nunjucks and returning `undefined` which was breaking the build, so I had to wrap it in a `{% raw %}`

Reply:
Oops, thanks for the catch here!